doc/index.rst

```diff
@@ -1,224 +1,42 @@
 :orphan:
 
-.. title:: Matplotlib: Python plotting
+.. title:: Matplotlib Documentation
 
 .. module:: matplotlib
 
-Matplotlib: Visualization with Python
--------------------------------------
+Matplotlib Documentation 
+------------------------
 
-Matplotlib is a comprehensive library for creating static, animated,
-and interactive visualizations in Python.
-
-.. raw:: html
-
-   <div class="responsive_screenshots">
-      <a href="tutorials/introductory/sample_plots.html">
-         <div class="responsive_subfig">
-         <img align="middle" src="_images/sphx_glr_membrane_thumb.png"
-          border="0" alt="screenshots"/>
-         </div>
-         <div class="responsive_subfig">
-         <img align="middle" src="_images/sphx_glr_histogram_thumb.png"
-          border="0" alt="screenshots"/>
-         </div>
-         <div class="responsive_subfig">
-         <img align="middle" src="_images/sphx_glr_contour_thumb.png"
-          border="0" alt="screenshots"/>
-         </div>
-         <div class="responsive_subfig">
-         <img align="middle" src="_images/sphx_glr_3D_thumb.png"
-          border="0" alt="screenshots"/>
-         </div>
-      </a>
-   </div>
-   <span class="clear_screenshots"></span>
+Welcome to the documentation for Matplotlib |release|.
 
 Matplotlib makes easy things easy and hard things possible.
```

Comment:
I still feel this is weak statement. One can say that about many libraries. And unless a library explicitly limits its scope that should be universal design goal: Give easy a access to standard functionality and have some lower-level APIs for advanced stuff.

I would remove this design goal as not particularly relevant here, but I know that other people are quite attached to it. So I won't fight about it. I just want to state my opinion here once again.

We should however keep the one-line description of what matplotlib *is* above:
```
Matplotlib is a comprehensive library for creating static, animated,
and interactive visualizations in Python.
```